OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp

```diff
@@ -260,6 +260,12 @@ bool GraphicsView::addComponent(QString className, QPointF position)
         pComponentInfo->setStartCommand("StartTLMOpenModelica");
       } else if (fileInfo.suffix().compare("in") == 0) {
         pComponentInfo->setStartCommand("StartTLMBeast");
+      } else if (fileInfo.suffix().compare("hmf") == 0) {
+        pComponentInfo->setStartCommand("StartTLMHopsan");
+      } else if (fileInfo.suffix().compare("in") == 0) {
```

Comment:
This looks like an typo. Shouldn't it be `else if (fileInfo.suffix().compare("fmi") == 0)`?